src/applications/appeals/testing/sc/content/facilityTypes.js

```diff
@@ -16,3 +20,24 @@ export const facilityTypeChoices = {
       'This includes providers who aren’t community care providers, and who don’t work at a military treatment facility. We’ll need to get your permission to get your medical records from this type of provider. Or you can upload these medical records yourself later in this application.',
   },
 };
+
+export const facilityTypeReviewField = ({ formData }) => {
+  const selected = Object.entries(formData).filter(([_key, value]) => value);
+  const list = readableList(
+    selected.map(
+      ([key]) =>
+        key === 'nonVa'
+          ? facilityTypeChoices[key].title
+          : facilityTypeChoices[key],
```

Comment:
You're not checking if `key` is in `facilityTypeChoices`, so you could end up with the dreaded `undefined` here.
Would it be better to refactor this to remove the `nonVa` check and do something like this instead (in quasi pseudo-code)?
```typescript
{
    if (key in facilityTypeChoices) {
        return facilityTypeChoices[key].title || facilityTypeChoices[key];
    } else {
        // log unknown `key` value
        return 'Unknown Facility Type Choice';
    }
}
```

Reply:
Ok, updated... I also forgot to include the "Other" text input in the list